apps/frontend/tests/component_tests/ViewListing.cy.tsx

```diff
@@ -326,38 +192,18 @@ describe("<ViewListing/>", () => {
             <Route path="/messages" element={<Messages />} />
           </Routes>
         </MemoryRouter>
-      </TestProviders>,
+      </TestProviders>
     );
-    const listingObject = {
-      title: "Genuine Unicorn Tears - Guaranteed to Add Sparkle to Your Life!",
-      description:
-        "Are you tired of mundane tears? Try our premium unicorn tears sourced straight from the enchanted forest of Eldoria! Each tear is carefully harvested under the light of a full moon and comes with a certificate of authenticity. Sprinkle them on your morning cereal for an extra magical start to your day!",
-      price: 20000,
-      seller_profile: { name: "Merlin the Wizard", userID: "A13097430" },
-      dateCreated: "1980-05-24T15:30:00Z",
-      distance: 420.1,
-      images: [
-        { url: "https://picsum.photos/200/300" },
-        { url: "https://picsum.photos/200/400" },
-        { url: "https://picsum.photos/200/300" },
-        { url: "https://picsum.photos/200/230" },
-        { url: "https://picsum.photos/200/100" },
-      ],
-      status: "AVAILABLE",
-      reviews: [
-        {
-          listing_review_id: "1",
-          reviewerName: "John Doe",
-          stars: 5,
-          comment: "This is the best product I have ever bought!",
-          userID: "1",
-          listingID: "1",
-          dateCreated: "2024-05-23T15:30:00Z",
-          dateModified: "2024-05-23T15:30:00Z",
-        },
-      ],
-    };
 
+    cy.wait("@getListing");
+
+    cy.get("#review_text").type("This is a great product!");
+
+    cy.get("button").contains("Post").click();
+    cy.contains("Please provide a star rating").should("be.visible");
+  });
+
+  it("should not allow you to delete another person's review", () => {
```

Comment:
I am sorry 🫣 I don't have a test for deleting your own reviews because I was really struggling with mocking your own userID so that you could only have the option to delete your reviews if the `reviewerID === yourUserID`. Tried this for several hours with no success. 

If people have suggestions please let me know and I would be happy to try :)

Reply:
Yeah, I just took a stab at it myself, didn't get very far, but I'm satisfied that with the manual tests I've done that it works properly